ColabTool/MultipleImagesPost/index.jsx

```diff
@@ -3,9 +3,19 @@ import {
   LinkifiedText,
   MultipleImages,
 } from '@bufferapp/components';
-import style from './style.css';
+// import style from './style.css';
```

Comment:
This is so amazing, @hharnisc! 😄  
We can probably remove this comment, I think.